plugins/kotlin/gradle/gradle-java/src/org/jetbrains/kotlin/idea/gradleJava/configuration/KotlinGradleCoroutineDebugProjectResolver.kt

```diff
@@ -9,7 +9,11 @@ import org.jetbrains.kotlin.idea.extensions.KotlinJvmDebuggerFacade
 import org.jetbrains.plugins.gradle.service.project.AbstractProjectResolverExtension
 
 class KotlinGradleCoroutineDebugProjectResolver : AbstractProjectResolverExtension() {
-    val log = Logger.getInstance(this::class.java)
+    companion object {
+        val log = Logger.getInstance(this::class.java)
+        private const val KOTLINX_COROUTINES_DEBUG_PROPERTY_NAME = "kotlinx.coroutines.debug"
```

Comment:
It'd be better to use const properties `kotlinx.coroutines.DEBUG_PROPERTY_NAME` and `kotlinx.coroutines.DEBUG_PROPERTY_VALUE_OFF` here instead of hardcoding their values.